tools/topology/topology2/include/pipelines/cavs/deepbuffer-playback.conf

```diff
@@ -47,28 +48,31 @@ Class.Pipeline."deepbuffer-playback" {
 			type	"aif_in"
 			node_type $HDA_HOST_OUTPUT_CLASS
 			num_input_audio_formats 3
-			num_output_audio_formats 3
+			num_output_audio_formats 1
 			num_output_pins 1
 			deep_buffer_dma_ms	$DEEPBUFFER_FW_DMA_MS
 			# 16-bit 48KHz 2ch
-			Object.Base.audio_format.1 {
-				out_bit_depth		32
-				out_valid_bit_depth	32
-			}
+			Object.Base.input_audio_format [
+				{
+					in_bit_depth		16
+					in_valid_bit_depth	16
+				}
+				{
+					in_bit_depth		32
+					in_valid_bit_depth	24
+				}
+				{
+					in_bit_depth		32
+					in_valid_bit_depth	32
+				}
+			]
 			# 24-bit 48KHz 2ch
-			Object.Base.audio_format.2 {
-				in_bit_depth		32
-				in_valid_bit_depth	24
-				out_bit_depth		32
-				out_valid_bit_depth	32
-			}
-			# 32-bit 48KHz 2ch
-			Object.Base.audio_format.3 {
-				in_bit_depth		32
-				in_valid_bit_depth	32
-				out_bit_depth		32
-				out_valid_bit_depth	32
-			}
+			Object.Base.output_audio_format [
+				{
+					out_bit_depth		32
+					out_valid_bit_depth	32
+				}
+			]
```

Comment:
the comments on line 54 and 69 are no longer relevant, remove?

Reply:
this patch was from a different PR, will fix it in a follow up